apps/chat/src/store/publication/publication.epics.ts

```diff
@@ -1097,6 +1097,26 @@ const approvePublicationEpic: AppEpic = (action$, state$) =>
             );
           }
 
+          const appResources = selectedPublication.resources.filter((r) =>
+            isApplicationId(r.targetUrl),
+          );
+          const appResourcesToUnpublish = appResources.filter(
+            (r) => r.action === PublishActions.DELETE,
+          );
```

Comment:
it can be done in one cycle
```suggestion
          const appResourcesToUnpublish = selectedPublication.resources.filter((r) =>
             r.action === PublishActions.DELETE && isApplicationId(r.targetUrl),
          );          
```